db/db_test.cc

```diff
@@ -2469,11 +2469,13 @@ class MultiThreadedDBTest : public DBTest,
 TEST_P(MultiThreadedDBTest, MultiThreaded) {
   anon::OptionsOverride options_override;
   options_override.skip_policy = kSkipNoSnapshot;
+  Options options = CurrentOptions(options_override);
   std::vector<std::string> cfs;
   for (int i = 1; i < kColumnFamilies; ++i) {
     cfs.push_back(ToString(i));
   }
-  CreateAndReopenWithCF(cfs, CurrentOptions(options_override));
+  Reopen(options);
```

Comment:
In some test cases of this test, the options are not compatible with parallel memtable inserts. Up to here. Before the test begins, the DB is opened with the default setting, which has parallel inserts on. Here we create an `options` which may contain an option that is conflicting with the parallel memtable inserts and we use that to add a column family. It will fail.

The solution here is to reopen the DB before adding the column family if the column family we are adding is not compatible with parallel memtable inserts.